extra space underneath last analysis

JorisGoosen opened the pull request:
it isnt always perfect but I think Ive got the bases mostly covered:
- on add analysis, remove and move
- on change analysis (results updated)
- on reRender

Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2089

JorisGoosen committed:
extra space underneath last analysis

it isnt always perfect but I think Ive got the bases mostly covered:
- on add analysis, remove and move
- on change analysis (results updated)
- on reRender

Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2089

JorisGoosen committed:
small fix

JorisGoosen commented on Desktop/html/js/analyses.js:
```diff
@@ -122,6 +130,23 @@ JASPWidgets.Analyses = JASPWidgets.View.extend({
+		if(this.analyses.length > 0)
+		{
+			var lastAnalysis = this.analyses[this.analyses.length - 1];
+			height = height - lastAnalysis.$el.outerHeight();
```
```suggestion
			height = Math.max(0, height - lastAnalysis.$el.outerHeight());
```

JorisGoosen committed:
Update Desktop/html/js/analyses.js

boutinb commented on Desktop/html/js/analyses.js:
```diff
@@ -122,6 +130,23 @@ JASPWidgets.Analyses = JASPWidgets.View.extend({
+
+	setBottomSpacerHeight: function() {
+		var bottomSpacer = $("#bottomSpacer");
+		var height		 = window.innerHeight;
```
Use better $(windows).innerHeight(): jQuery takes better into account the scrollbars.
I have noticed that when the horizontal scrollbar appears, then the bottomSpacer becomes a bit too high.

JorisGoosen replied:
```suggestion
		var height		 = $(windows).innerHeight();
```

like this?

JorisGoosen committed:
Bruno mentioned it is better to use `$(window).innerHeight()`.

> jQuery takes the scrollbars into account better.
> I have noticed that when the horizontal scrollbar appears, then the bottomSpacer becomes a bit too high.
